core/templates/components/state-directives/answer-group-editor/answer-group-editor.directive.html

```diff
@@ -66,7 +66,7 @@
       If the learner's answer...
     </strong>
     <ul class="nav oppia-option-list nav-stacked nav-pills" role="tablist" ng-model="$ctrl.rules">
-      <li ng-repeat="rule in $ctrl.rules" ng-class="{'active': $ctrl.activeRuleIndex === $index} $last ? oppia-answer-rule-block : {}" class="mt-0 oppia-rule-block oppia-sortable-rule-block protractor-test-rule-block">
+      <li ng-repeat="rule in $ctrl.rules" ng-class="{'active': $ctrl.activeRuleIndex === $index} {'oppia-answer-rule-block': $last}" class="mt-0 oppia-rule-block oppia-sortable-rule-block protractor-test-rule-block">
```

Comment:
@Hudda This should all be in a single dict, right? Look at other instances of ng-class, I've never seen space separated dicts like this.

Please double-check by setting 'active' or 'oppia-answer-rule-block' to do something obvious visually (e.g. have the colour be black, or the border be huge and yellow, or something) and check that you can actually trigger both conditions while playing with the page.